config/crd/patches/cainjection_in_processingjobs.yaml

```diff
@@ -0,0 +1,8 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
+# CRD conversion requires k8s 1.13 or later.
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
Regarding [webhook conversion](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#webhook-conversion). I think its used for upgrades or converting objects between versions if we have later

and [CA Injection](https://cert-manager.io/docs/concepts/ca-injector/) is used for authentication with the webhook API server

These are not in other CRDs probably because kubebuilder was not used to create them? There were created when I used the `kubebuilder create api`